modules/java_api/tests/PrePostProcessorTests.java

```diff
@@ -17,15 +16,12 @@ public class PrePostProcessorTests extends OVTest {
     public void setUp() {
         core = new Core();
         net = core.read_model(modelXml);
-        float[] data = {
-            0.0f, 1.1f, 2.2f, 3.3f, 4.4f, 5.5f, 6.6f, 7.7f, 8.8f, 9.9f, 1.1f, 2.2f, 3.3f
-        };
+        float[] data = {0.0f, 1.1f, 2.2f, 3.3f, 4.4f, 5.5f, 6.6f, 7.7f, 8.8f, 9.9f, 1.1f, 2.2f};
         input = new Tensor(dimsArr, data);
     }
 
-    @Ignore
```

Comment:
Do you know if this test became stable? Of not, let's enable ot by a separate PR

Reply:
Yes, I think the problem was about `TensorFloat` method